lib/cose/key/rsa.rb

```diff
@@ -133,8 +135,14 @@ def self.keyword_arguments_for_initialize(map)
 
       private
 
+      def private?
+        [d, p, q, dp, dq, qinv].none?(&:nil?)
+      end
+
       def bn(data)
-        OpenSSL::BN.new(data, 2)
+        if data
```

Comment:
This conditional is needed because of https://github.com/ruby/openssl/issues/248